.github/workflows/no-response.yml

```diff
@@ -18,13 +18,15 @@ jobs:
       contents: read
       issues: write
     steps:
-      - uses: lee-dohm/no-response@9bb0a4b5e6a45046f00353d5de7d90fb8bd773bb # v0.5.0
+      - uses: MBilalShafi/no-response-add-label@9657b5bc1e09373e571df54429c3c167edaad556
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           # Number of days of inactivity before an Issue is closed for lack of response
           daysUntilClose: 7
           # Label requiring a response
-          responseRequiredLabel: 'status: waiting for follow-up'
+          responseRequiredLabel: "status: waiting for author's response"
+          # Label to add back when required label is removed
+          optionalFollowupLabel: "status: waiting for maintainer's response"
```

Comment:
Shorter?

```suggestion
          optionalFollowupLabel: "status: waiting for maintainer"
```

```suggestion
          optionalFollowupLabel: "status: need maintainer's response"
```

Reply:
My proposition in the notion doc is:
`status: waiting for maintainer`
`status: waiting for author`

Which is shorter but still clear.

Reply:
Looks good, implemented the change!

Reply:
> status: waiting for author

Could this be confused with "author" as the author of the code?

Would "waiting for user" be clearer? As I understand it, the GitHub Action nor maintainers truly need the author to answer, but more have confirmation from users.